generic/tests/cfg/vlan.cfg

```diff
@@ -13,7 +13,9 @@
     kill_vm_vm2 = yes
     kill_vm_gracefully_vm2 = no
     cmd_type = ip
-    msg_pattern = 'RTNETLINK answers: Numerical result out of range'
+    msg_pattern = '8021q: Invalid VLAN id'
+    RHEL.7:
```

Comment:
Do we consider RHEL.6? Otherwise should have a `no RHEL.6` variant.

Reply:
@vivianQizhu Thanks, will update it later.

Reply:
@vivianQizhu Updated, please help look again, thanks.